app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less

```diff
@@ -200,6 +200,21 @@
             }
         }
     }
+    
+    .toolbar {
+        &.wishlist-toolbar {
+            .limiter {
+                float: none;
+            }
```

Comment:
IMO you can use flexbox for arrange layout easier than this

Reply:
Example

`.toolbar {
        &.wishlist-toolbar {
            .pager {
                .lib-vendor-prefix-display();
                align-items: center;
                flex-wrap: wrap;
                justify-content: space-between;
            }
        }
    }`